<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomFileDialogClass</class>
 <widget class="QWidget" name="CustomFileDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CustomFileDialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{font-family: Microsoft YaHei;}

QWidget#CustomFileDialogClass
{
    border-image: url(:/CustomFileDialog/Resources/back.png);
    border-width: 45px 8px 8px 8px;
}

QPushButton#sysClose {
   border-image: url(:/res_main/t_12.png); 
}
QPushButton#sysClose:hover {
   border-image: url(:/res_main/t_13_hover.png);
}
QPushButton#sysClose:pressed {
   border-image: url(:/res_main/t_13_hover.png); 
}

QPushButton#sysMin {
   border-image: url(:/CustomFileDialog/Resources/sysmin.png); 
}
QPushButton#sysMin:hover {
   border-image: url(:/CustomFileDialog/Resources/sysmin_hover.png); 
}
QPushButton#sysMin:pressed {
   border-image: url(:/CustomFileDialog/Resources/sysmin_hover.png); 
}

QPushButton#sysClose {
   border-image: url(:/CustomFileDialog/Resources/sysclose.png); 
}
QPushButton#sysClose:hover {
   border-image: url(:/CustomFileDialog/Resources/sysclose_hover.png); 
}
QPushButton#sysClose:pressed {
   border-image: url(:/CustomFileDialog/Resources/sysclose_hover.png); 
}

QLabel{
	font-size:12px;
	color:#666666;
}

QComboBox::down-arrow{
image: url(:/CustomFileDialog/Resources/drop-down.png);padding-right:4px;
}
QComboBox::down-arrow:on{
image: url(:/CustomFileDialog/Resources/drop-up.png);
padding-right:4px;
}
QComboBox::drop-down{
background-color:rgb(255,255,255,0);
}
QComboBox::drop-down:disabled,QComboBox:disabled{
background-color:#f8f8f8
}
QLineEdit,QComboBox{
color: #666666;border: 1px solid #DDDDDD;border-radius:2px;
}
QListView, QTreeView{
outline: 0px;background-color:white;border: 1px solid #DDDDDD;color:#666666;selection-color:#666666;selection-background-color: #f6f6f6;border-radius:2px;
}
QListView::item, QTreeView::item{
height:24px;
text-align:center;
}
 QScrollBar:horizontal{
     background: #ffffff;
     height: 6px;
 }
 QScrollBar::handle:horizontal{
     background:#cccccc;
     border-radius:3px;
     height: 6px;
 }

QSplitter::handle {
	background-color: rgb(255, 255, 255);
}

 QScrollBar::sub-line:horizontal{
     background:transparent;
 }
 QScrollBar::add-line:horizontal{
     background:transparent;
 }

 QScrollBar:vertical {
     background: #ffffff;
     width: 6px;
 }
 QScrollBar::handle:vertical {
     background:#cccccc;
     border-radius:3px;
     width: 6px;
 }

 QScrollBar::sub-line:vertical {
     background:transparent;
 }
 QScrollBar::add-line:vertical {
     background:transparent;
 }

QHeaderView::section {    
    color: #333333;  
	height:24px;
    font-size:12px;  
    background-color: #f8f8f8;  
    border:1px solid #eeeeee;  
    border-left:none;
	border-top:none;  
	padding:6px 10px 1px 10px;
}  

QHeaderView::down-arrow {
    image: url(:/CustomFileDialog/Resources/drop-down.png);
}

QHeaderView::up-arrow {
    image: url(:/CustomFileDialog/Resources/drop-up.png);
} 
QMenu{background-color:white;border-style:none;}
QMenu::item {font-size: 12px;color: #666666;
background-color:#ffffff;padding:8px;}
QMenu::item:selected {background-color:#f6f6f6;color:#666666;}
QMenu::item:!enabled {background-color:white;color:#999999;}
QMenu::item:hover {color:#666666;}
QMenu::separator {
    height: 1px;
    background: #eeeeee;
    margin: 2px 5px 2px 10px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>7</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>36</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	font-size:14px;
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sysMin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="toolTip">
        <string>最小化</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sysClose">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>23</width>
         <height>23</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
       <property name="toolTip">
        <string>关闭</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="bodyWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="bodylayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="customfiledialog.qrc"/>
 </resources>
 <connections/>
</ui>
